x/mongo/driver/operation.go

```diff
@@ -435,6 +435,9 @@ func (op Operation) Execute(ctx context.Context, scratch []byte) error {
 				if err.Code == 64 || err.Code == 50 || tt.WriteConcernError.Retryable() {
 					err.Labels = []string{UnknownTransactionCommitResult}
 				}
+				if tt.WriteConcernError.Retryable() {
```

Comment:
This code will only run if we're committing a transaction. I think we can move this to the very top of the case near line 400 where we call `tt.Retryable` because `WriteCommandError.Retryable` just checks if the inner write concern error is retryable.

So this could be something like:

```
case WriteCommandError:
// unsupported storage engine check
isRetryableWriteError := tt.Retryable()
if isRetryableWriteError { // add label }
// rest of logic with tt.Retryable calls replaced with isRetryableWriteError variable
```

Reply:
As is, I can't do it there since WriteCommandErrors don't have labels. What I'm doing right now is adding the label in situations where we have an Error object, which has labels. Do we want add a Labels field to WriteCommandErrors?